pyonfx/timestamps.py

```diff
@@ -0,0 +1,552 @@
+# -*- coding: utf-8 -*-
+# PyonFX: An easy way to create KFX (Karaoke Effects) and complex typesetting using the ASS format (Advanced Substation Alpha).
+# Copyright (C) 2019 Antonio Strippoli (CoffeeStraw/YellowFlash)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PyonFX is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see http://www.gnu.org/licenses/.
+from __future__ import annotations
+from enum import Enum
+import json
+import os
+import re
+import shutil
+import subprocess
+import sys
+import warnings
+from decimal import Decimal
+from fractions import Fraction
+from io import StringIO, TextIOWrapper
+from typing import Callable, List, Optional, Tuple, Union
+
+
+class RoundingMethod(Enum):
+    FLOOR: Callable[[Fraction], int] = lambda ms: int(ms)
+    ROUND: Callable[[Fraction], int] = lambda ms: int(ms + Fraction("0.5"))
```

Comment:
There is no need to define them as callable, as we never call them. Simply keeping them as enum should be enough.

Reply:
We call them.

See:
- [convert.py - line 206](https://github.com/CoffeeStraw/PyonFX/blob/242f7a4d87bc283f807c03b898a49616203d0ed6/pyonfx/convert.py#L206)
- [timestamps.py - line 184](https://github.com/CoffeeStraw/PyonFX/blob/242f7a4d87bc283f807c03b898a49616203d0ed6/pyonfx/timestamps.py#L184)
- [timestamps.py - line 189](https://github.com/CoffeeStraw/PyonFX/blob/242f7a4d87bc283f807c03b898a49616203d0ed6/pyonfx/timestamps.py#L189)
- [timestamps.py - line 193](https://github.com/CoffeeStraw/PyonFX/blob/242f7a4d87bc283f807c03b898a49616203d0ed6/pyonfx/timestamps.py#L193)
- [timestamps.py - line 242](https://github.com/CoffeeStraw/PyonFX/blob/242f7a4d87bc283f807c03b898a49616203d0ed6/pyonfx/timestamps.py#L242)
- [timestamps.py - line 474](https://github.com/CoffeeStraw/PyonFX/blob/242f7a4d87bc283f807c03b898a49616203d0ed6/pyonfx/timestamps.py#L474)